linera-service/src/grpc_proxy.rs

```diff
@@ -257,7 +291,15 @@ impl ValidatorNode for GrpcProxy {
         request: Request<ChainInfoQuery>,
     ) -> Result<Response<ChainInfoResult>, Status> {
         let (mut client, inner) = self.client_for_proxy_worker(request).await?;
-        client.handle_chain_info_query(inner).await
+        match client.handle_chain_info_query(inner).await {
+            Ok(chain_info_result) => {
+                PROXY_REQUEST_SUCCESS
+                    .with_label_values(&["handle_chain_info_query"])
+                    .inc();
+                Ok(chain_info_result)
+            }
+            Err(status) => Err(status),
+        }
     }
 
     #[instrument(skip_all, err(Display))]
```

Comment:
<i class='graphite__hidden'>[Re: lines 306 to 306]<br/></i>

I'm right in assuming we don't want to log metrics on this one, right?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1171?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Synched on Slack, will add this later